src/locales/en/translation.json

```diff
@@ -209,6 +209,9 @@
     "campaignHeader": "Light Up Chinatown Fundraising Campaign:",
     "campaignDates": "November 1 - 30, 2020",
     "campaignDaysLeft": "More Days Left",
+    "progressBarHeader": "Total Raised: ",
+    "raiseGoal1": " GOAL 1: $23.5k",
+    "raiseGoal2": " GOAL 2: $47k",
```

Comment:
mind adding into the chinese version of the file too? (no need to translate it, just the english is fine)